hepcrawl/parsers/jats.py

```diff
@@ -344,10 +359,12 @@ def get_author_affiliations(self, author_node):
 
         return affiliations
 
-    @staticmethod
-    def get_author_emails(author_node):
+    def get_author_emails(self, author_node):
         """Extract an author's email addresses."""
         emails = author_node.xpath('.//email/text()').extract()
+        referred_ids = author_node.xpath('.//xref[@ref-type="aff"]/@rid').extract()
+        aff_emails = [self.get_emails_from_refs(rid) for rid in referred_ids]
+        emails = sum(aff_emails, emails)
```

Comment:
`emails.extend(aff_emails)`

Reply:
It's more elegant this way, however I don't think this will work as `self.get_emails_from_refs` returns a list of strings. In theory there could be more than one email address in an affiliation field, as you said the spec is quite lax. So the question is do we need that, or we can just extract one and then this solution is fine?